addons/web/static/src/xml/translation_dialog.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <div t-name="TranslationDialog" class="translation_dialog">
+        <div class="row" t-foreach="widget.data" t-as="term">
+            <div class="col-3 language">
+                <t t-esc="term.langName"></t>
+            </div>
+            <div class="col-9 translation">
```

Comment:
i'd set col-12 for both, and col-lg-3/9, so that it's looking good in mobile (+ an higher margin-bottom on .row)